README.md

```diff
@@ -81,34 +78,48 @@ GH_TOKEN=${GH_TOKEN}
 # Branches
 TMS_BRANCH=main
 ED_BRANCH=main
+RULE_EXECUTER_BRANCH=main
 TP_BRANCH=main
 TADP_BRANCH=main
-RULE_901_BRANCH=main
+NATS_UTILITIES_BRANCH=main
+CMS_BRANCH=main
+BATCH_PPA_BRANCH=main
+ADMIN_BRANCH=main
+AUTH_SERVICE_BRANCH=main
+EVENT_FLOW_BRANCH=main
 
 # Ports
-TMS_PORT=5000
+TMS_PORT=3000
```

Comment:
Ditto - the current postman testing environments for the docker deployment all point to port 5000...